File: be_hotel/src/main/java/be_hotel/serive/account/AccountService.java
package be_hotel.serive.account;

import be_hotel.Modal.entity.Account;
import be_hotel.Modal.request.AccountRequest;
import be_hotel.Modal.request.LoginRequest;
import be_hotel.Modal.response.AuthReponse;
import org.springframework.data.domain.Page;

public interface AccountService {
    public Account createAccount(AccountRequest accountRequest);

    public Page getAccount(int page, int size);

    public Account updateAccount(int id,AccountRequest accountRequest);

    public String deleteAccount(Integer id);

    public AuthReponse login(LoginRequest loginRequest);
}

File: src/services/AccountService.js
import api from './api';

const AccountService = {
    getAccount(page,size){
        return api.get(`getAllAccount/${page}/${size}`)
    },
    createAccount(body){
        return api.post("createAccount", body)
    },
    updateAccount(id, body){
        return api.put("updateAccount/"+id, body)
    },  
    deleteAccount(id){
        return api.delete("deleteAccount/"+id)
    }
}

export default AccountService;